packages/frontend/src/components/RefreshButton.tsx

```diff
@@ -51,6 +51,21 @@ export const RefreshButton: FC<{ size?: MantineSize }> = memo(({ size }) => {
 
     useHotkeys([['mod + enter', onClick, { preventDefault: true }]]);
 
+    useEffect(() => {
+        const handleKeyDown = (event: KeyboardEvent) => {
+            if ((event.metaKey || event.ctrlKey) && event.key === 'Enter') {
+                event.preventDefault();
+                onClick();
+            }
+        };
+
+        document.addEventListener('keydown', handleKeyDown);
+
+        return () => {
+            document.removeEventListener('keydown', handleKeyDown);
+        };
+    }, [onClick]);
```

Comment:
It's the hotkey library default behaviour that ignores the keys when a input (and other elements) are in focus.
Check the lib documentation to learn how to disable this. https://mantine.dev/hooks/use-hotkeys/